sQuick.package/MainInterface.class/class/startUp.st

```diff
@@ -1,5 +1,4 @@
 as yet unclassified
 startUp
-	super startUp.
```

Comment:
In general, you don't send to super from the class side unless you have a specific reason. Also, if you want to hook into the system's startup, you have to register #addToStartUpList: in a class-side #initialize